src/elements/Segment/SegmentGroup.js

```diff
@@ -0,0 +1,42 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+} from '../../utils/propUtils'
+import _Segment from './_Segment'
+
+function SegmentGroup(props) {
+  const {
+    children, className,
+  } = props
+
+  const classes = cx(
+    className
+  )
+
+  const rest = getUnhandledProps(SegmentGroup, props)
+
+  return (
+    <_Segment className={classes} {...rest} _sdClass='sd-segment-group' _segmentClass='segments'>
+      {children}
+    </_Segment>
```

Comment:
A SegmentGroup only supports `horizontal`, `raised`, `stacked` and `piled` classes.  Whereas the individual Segment will not support these.

Because of this, I think we should just use a `div` here in the SegmentGroup and move those props from the _Segment to the SegmentGroup.  Then, rename the _Segment to Segment.

***

As an aside it is worth noting how the docs work here.  Any non-private class has documentation generated for it.  The doc blocks on props generate the props table, the class/function doc block generates the primary component heading.

This means if a component is marked private by an underscore, it will not show up in the docs.  This also means, if another component imports a private a component and uses it (Segment => _Segment), the exposed component (Segment) will not appear to have props or description documentation because the docs are hidden in the private component.